### PR TITLE
Arrange Status Display

### PR DESCRIPTION
**Pull request by ab7pa**
Arrange the Status display and add default value for disabled meshrf.

### CODE REVIEW

File: components/MeshRF.vue [hunk 1]
```diff
@@ -3,21 +3,21 @@
...
     </v-card-title>
     <v-card-text>
       <v-row>
         <v-col class="text-end">
           <p class="label">SSID:</p>
           <p class="label">Channel:</p>
-          <p class="label">Bandwidth:</p>
-          <p class="label">Frequency:</p>
+          <p class="label">Bandwidth (MHz):</p>
+          <p class="label">Frequency (MHz):</p>
```
Comment (dman776) [inline components/MeshRF.vue:14]: Frequency is returned from the API with the GHz label already.  (It's not in MHz)

**Review by dman776 — CHANGES_REQUESTED** [review]
see comment on Frequency label.  All other changes look fine.

File: components/MeshRF.vue (hunk 1 above)
Comment (ab7pa) [reply components/MeshRF.vue:14]: Perfect!  Thanks.  Just pushed that change.

Comment (dman776) [reply components/MeshRF.vue:14]: actually, i think it's better to update the api to return the raw freq in MHz.   I just submitted that PR for discussion.  Let's hold this until that is resolved.   If it is approved, then the (MHz) label will be fine.

**Comment by dman776** [comment]
depends on the outcome of aredn repo PR #101

**Comment by ab7pa** [comment]
Good idea.  Changed _frequency_ units to MHz.

**Review by dman776 — APPROVED** [review]
Looks great

**Pull request merged**